tailwind.config.js

```diff
@@ -5,19 +5,31 @@ const defaultTheme = require('tailwindcss/defaultTheme');
 export default {
 	content: ['./src/**/*.{js,jsx,ts,tsx}'],
 	theme: {
-		colors: {
-			eggshell: '#FDFDF2',
-			sage: '#BBC893',
-			brown: '#D4A373',
-			yellow: '#FEFAE0',
-			vanilla: '#FAEDCD',
-			'pale-green': '#E9EDC9',
+		screens: {
```

Comment:
praise: thank you for adding these in, we can use them in the home page as well as navbar to fine tune things, too